src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,15 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  categories: [],
+};
+const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState,
+  reducers: {
+    checkStatus: () => 'Under construction',
+  },
```

Comment:
- [ ] @Chandan-devs-tech Great work on your categories reducer, however, the reducer needs to check the status if it matches the string "Under construction" and then return "Under construction" string.
```suggestion
  reducers: {
   checkStatus: (state, action) => {
      state.categories =
        action.payload === "Under construction"
          ? "Under construction"
          : state.categories
    },
  },
```